website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -509,6 +509,12 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
+		component_discarded_events_total: {
```

Comment:
👍

Reply:
Updated upgrade guide in dddf9c7. Let me know how it looks